snuba/replacer.py

```diff
@@ -589,6 +591,12 @@ def _attempt_emitting_metric_for_projects_exceeding_limit(
         projects_exceeding_limit = (
             self.__processing_time_counter.get_projects_exceeding_limit()
         )
+        # this is for testing purely on S4S, will delete immediately after testing
+        if 2 not in projects_exceeding_limit and get_config(
```

Comment:
It’s hard to determine how many replacement operations I have to manually perform to reach the threshold. One way to get around this is to always add S4S (project id = 2) to the list of offending projects, so effectively skipping all replacements for S4S. After verifying this works, I’ll have to immediately merge a PR that undoes this. What do people think?

Reply:
We decided to make `COUNTER_WINDOW_SIZE` configurable through the runtime config, so by setting `counter_window_size_minutes` = 0 in the S4S runtime config, we are effectively skipping every replacements project in S4S only